### ISSUE TITLE
Tournament startDate is always invalid when creating a new tournament

### ISSUE THREAD

**Issue by minimalsm** [sjohnston00/HotshotTournaments#14]
![Screenshot 2021-03-19 at 16 52 09](https://user-images.githubusercontent.com/62268199/111816462-c9803900-88d4-11eb-9c42-658d46ecd2fe.png)

startDate currently is set to the current time (HH:MM:SS). This means that even if the form is filled out instantly and submitted it will start trigger the validation error seen above.

https://github.com/sjohnston00/HotshotTournaments/blob/c66869553040ca75938f6a24d2098e2cd3786895/src/controllers/tournamentsController.js#L67-L81

**Comment by sjohnston00** [comment]
Yeah its quick fix, just set the HH:MM:SS to zero so its the start of that day

**Comment by minimalsm** [comment]
@sjohnston00 wouldn't it need to be the following day? the current problem is that the default date is trying to set an event that has already started.

**Comment by sjohnston00** [comment]
I don't know, maybe you want to set up a tournament for that night, maybe its 3pm and you want to set up a tournament with your pals for 9pm. 
Maybe instead of letting the user create a tournament that has already started (e.g it's 3pm and you create a tournament that starts at 11am that same day), maybe only allow tournaments to be created withing the next hour (e.g it's 3pm and the earliest start time a new tournament is 4pm).

**Comment by sjohnston00** [comment]
Is there a way to have the HTML 5 datetimelocal picker only allow intervals of 15mins, instead of allowing users to set times like startDate: 21/05/2021 15:22 they would only have available 15:00, 15:15, 15:30, 15:45 ....
